src/test/java/io/pravega/connectors/flink/FlinkPravegaSchemaRegistryReaderTestITCase.java

```diff
@@ -51,19 +53,62 @@ public MyTest(String a) {
         }
     }
 
+    @SuppressWarnings("checkstyle:ConstantName")
```

Comment:
We don't need to keep the constant name the same with the avro-generated class, a constant named `SCHEMA` would be enough. the class name can also be simplified to `User` as well.
Another path is that we keep this class real avro generated, we need to have a `.avsc` file in the `resources` folder and let avro generate the class. See 

https://github.com/apache/flink/blob/master/flink-formats/flink-avro-glue-schema-registry/src/test/java/resources/avro/user.avsc
https://github.com/apache/flink/blob/master/flink-formats/flink-avro-glue-schema-registry/src/test/java/org/apache/flink/formats/avro/glue/schema/registry/User.java

Both approaches are good to me.

Reply:
agreed, will fix this